src/app/pages/wallet/wallet.component.html

```diff
@@ -1,24 +1,22 @@
-<div *ngIf="actor; else noSession">
-    <h2>{{ actor }}</h2>
+<div *ngIf="sessionService.currentSession?.actor; else noSession">
+    <h2>{{ sessionService.currentSession.actor }}</h2>
```

Comment:
Here, the previous lines were correct. No need to change them:

```suggestion
<div *ngIf="actor; else noSession">
    <h2>{{ actor }}</h2>
```